CRM/Contribute/Form/Contribution.php

```diff
@@ -281,7 +281,7 @@ public function preProcess() {
         $lineItem = CRM_Price_BAO_LineItem::getLineItems($this->_compId);
       }
       else {
-        $lineItem = CRM_Price_BAO_LineItem::getLineItems($this->_id, 'contribution', 1, TRUE, TRUE);
+        $lineItem = CRM_Price_BAO_LineItem::getLineItemsByContributionID($this->_id, TRUE);
```

Comment:
We should try to eliminate the 'TRUE' hear or leave it unchanged for now & add a code comment explaining